conductor.json

```diff
@@ -379,7 +379,13 @@
         "DATABASE_PASSWORD": "{{env.DATABASE_PASSWORD}}",
         "DATABASE_HOST": "{{services.postgres.host}}",
         "INTERVAL": "10000",
-        "METRICS_SERVICE": "http://{{services.asl-metrics.host}}:{{services.asl-metrics.port}}"
+        "METRICS_SERVICE": "http://{{services.asl-metrics.host}}:{{services.asl-metrics.port}}",
+        "KEYCLOAK_REALM": "{{env.KEYCLOAK_REALM}}",
+        "KEYCLOAK_URL": "{{env.KEYCLOAK_URL}}",
+        "KEYCLOAK_CLIENT": "{{env.KEYCLOAK_CLIENT}}",
+        "KEYCLOAK_SECRET": "{{env.KEYCLOAK_SECRET}}",
+        "KEYCLOAK_USERNAME": "{{env.KEYCLOAK_USERNAME}}",
+        "KEYCLOAK_PASSWORD": "{{env.KEYCLOAK_PASSWORD}}"
```

Comment:
We may need to use different values from the `.env` to these since these refer to the admin user defined to perform email changes in resolver, which will not be the same user that has access to metrics.

Reply:
Now using `KC_EXPORTS_USERNAME` and `KC_EXPORTS_PASSWORD`